qiskit/algorithms/amplitude_estimators/ae.py

```diff
@@ -348,6 +355,13 @@ def estimate(self, estimation_problem: EstimationProblem) -> "AmplitudeEstimatio
         if estimation_problem.objective_qubits is None:
             raise ValueError("The objective_qubits property of the estimation problem must be set.")
 
+        if estimation_problem.has_good_state:
+            warnings.warn(
+                "The AmplitudeEstimation class does not support an is_good_state function to "
+                "identify good states. The oracle has to be set using the grover_operator, otherwise "
+                "a good state is identified by the objective qubits being in state 1."
```

Comment:
>"The oracle has to be set using the grover_operator, otherwise "
>"a good state is identified by the objective qubits being in state 1.

I am trying to figure `otherwise` in that sentence. Should if I replace it by "where" ie

>"The oracle has to be set using the grover_operator, where "
>"a good state is identified by the objective qubits being in state 1.

Is this the same. Maybe the otherwise is trying to say it has some fixed good state determination that is other than (i.e. not) determined by the good_state function which is not supported and ignored. Maybe we should add "ignored" too.

So perhaps?...

The is_good_state function will be ignored. The AmplitudeEstimation class does 
not support an is_good_state function to identify good states. The oracle has to
be set using the grover_operator where a good state is identified by the objective
qubits being in state 1.

Reply:
"Otherwise" is meant as in: If you want to set a custom oracle, you have to set it via the `grover_operator`. Otherwise (i.e. if no custom oracle is used) a the default oracle is used, which identifies good states as states, where all objective qubits are 1.

Reply:
I tried to make this clearer in 15b977a